doc/source/how-to/continuous-integration.rst

```diff
@@ -176,3 +176,354 @@ Workflow examples are provided for checking :ref:`Coding style`,
+  |  Read team discussions                                                              |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **admin:enterprise**                                                                | |:x:|                  |
+  |  Full control of enterprises                                                        |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **manage_runners:enterprise**                                                       | |:x:|                  |
+  |  Manage enterprise runners and runner groups                                        |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **manage_billing:enterprise**                                                       | |:x:|                  |
+  |  Read and write enterprise billing data                                             |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:enterprise**                                                                 | |:x:|                  |
+  |  Read enterprise profile data                                                       |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **audit_log**                                                                       | |:x:|                  |
+  |  Full control of audit log                                                          |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **read:audit_log**                                                                  | |:x:|                  |
+  |  Read access of audit log                                                           |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **codespace**                                                                       | |:x:|                  |
+  |  Full control of codespaces                                                         |                        |
+  +-------------------------------------------------------------------------------------+------------------------+
+  | **codespace:secrets**                                                               | |:x:|                  |
+  |  Ability to create, read, update, and delete codespace secrets                      |                        |
```

Comment:
Here codespace secrets is not capitalized or bold. Yes!

Reply:
Because it is a copy paste from somewhere else XD